bin/merge_and_align_libraries_from_easypqp.py

```diff
@@ -176,12 +180,15 @@ def concatenate_without_alignment(files):
        other_df = pd.read_csv(other, sep='\t')
 
        cols=[c for c in reference.columns if not 'NormalizedRetentionTime' in c and not 'TransitionId' in c]
+
+       seqs_ref=reference['ModifiedPeptideSequence'].values.tolist()
+       other_df = other_df[~other_df['ModifiedPeptideSequence'].isin(seqs_ref)]
        concat = pd.concat([reference,other_df])
-       regrouped = concat.iloc[concat[cols].drop_duplicates().index]
+       regrouped = concat #.iloc[concat[cols].drop_duplicates().index]
 
        reference = regrouped
 
-    combined_lib = regrouped.drop_duplicates()
+    combined_lib = regrouped #.drop_duplicates()
```

Comment:
what about the commented out part here and above?

Reply:
changed